resources/scripts/worker.js

```diff
@@ -1,5 +1,3 @@
-importScripts('fuse.min.js');
```

Comment:
@ayuhsya - Or you can just restore the `fuse.min.js` file back, and I'll merge this. Maybe we can work on loading `fuse.min.js` on another PR. :smile: